### PR TITLE
NO-ISSUE Allow @pkliczewski's team to lgtm PRs

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,6 +6,7 @@ filters:
       - code-reviewers
     reviewers:
       - code-reviewers
+      - reviewers-only
```
Comment (osherdp) [inline OWNERS:9]: bad naming
I think we should do ``code-approvers`` and ``code-reviewers`` and put ``code-approvers`` under ``approvers`` and under ``reviewers``

**Review by osherdp — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (YuviGold) [reply OWNERS:9]: Or maybe separate into actual teams like `core` and `edge`, etc

**Comment by jordigilh** [comment]
@pkliczewski can you rebase?

File: OWNERS (hunk 1 above)
Comment (jordigilh) [reply OWNERS:9]: `reviewers` sounds good to me, unless you expect more teams to join and be granted different roles. By the way, are we following a naming convention based on function or team's name? I think function makes more sense in the long run, but just my $0.02 :smile:

**Comment by jordigilh** [comment]
> @jordigilh have a look at #212, let me know what you think

looks good to me. Thanks for taking care of it so swiftly :smile:

**Comment by osherdp** [comment]
don't worry about it :)

**Pull request closed without merging**